apps/coscrad-frontend/src/components/term-of-the-day/term-of-the-day.presenter.tsx

```diff
@@ -0,0 +1,61 @@
+import { ICategorizableDetailQueryResult, ITermViewModel } from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import { Card, Divider } from '@mui/material';
+
+type HasTermOfTheDay = {
+    termOfTheDay: any
+}
+
+export const TermOfTheDayPresenter = ({ termOfTheDay }: HasTermOfTheDay, {
+    id,
+    term,
+    termEnglish,
+    contributor,
+    audioURL,
+}: ICategorizableDetailQueryResult<ITermViewModel>): JSX.Element => {
+
+    const today: Date = new Date();
```

Comment:
Break out a helper that takes care of this logic around determining the date.

Reply:
Resolved.